File: python/post_opentypeextension_2.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("POST", "/v1.0/me/messages(%27AAMkAGE1M2IyNGNmLTI5MTktNDUyZi1iOTVl===%27)/extensions", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: java/post_opentypeextension_2.java
HttpResponse<String> response = Unirest.post("https://graph.microsoft.com/v1.0/me/messages(%27AAMkAGE1M2IyNGNmLTI5MTktNDUyZi1iOTVl===%27)/extensions")
  .header("content-type", "application/json")
  .body("{ }")
  .asString();
